cmd/relayd/setting/setting.go

```diff
@@ -1,12 +1,26 @@
 package setting
 
 import (
+	"os"
+
 	"github.com/stratosnet/sds/utils"
 )
 
 var Config *config
 var HomePath string
 
+const (
+	VERSION     = "v0.8.1"
+	APP_VER     = 8
+	MIN_APP_VER = 8
```

Comment:
Please change the version number here. Line 13 to 15.

Reply:
changed to version 9